packages/sitecore-jss-react/src/components/Placeholder.tsx

```diff
@@ -73,15 +123,27 @@ class PlaceholderComponent extends PlaceholderCommon<PlaceholderComponentProps>
       renderingData,
       this.props.name
     );
+
+    const isEmptyPlaceholder = placeholderData.every(
+      (rendering: ComponentRendering | HtmlElementRendering) => isRawRendering(rendering)
+    );
+
     const components = this.getComponentsForRenderingData(placeholderData);
 
-    if (
-      this.props.renderEmpty &&
-      placeholderData.every((rendering: ComponentRendering | HtmlElementRendering) =>
-        isRawRendering(rendering)
-      )
-    ) {
-      return this.props.renderEmpty(components);
+    if (isEmptyPlaceholder) {
+      if (
+        typeof window !== 'undefined' &&
```

Comment:
Should be able to remove this window check - HorizonEditor.isActive() should handle already.

Reply:
HorizonEditor.isActive doesn't handle it correct, need to check type of `window`. Added comment in your PR

Reply:
Good catch, fixed in my PR.